site/content/api/Tenant-API.md

```diff
@@ -284,6 +284,7 @@ The table below provides an overview of the standard members defined for the JSO
 | *enabled*                | *no*      | *boolean*     | `true`       | If set to `false` the tenant is currently disabled. Protocol adapters MUST NOT allow devices of a disabled tenant to connect and MUST NOT accept data published by such devices. |
 | *trusted-ca*             | *no*      | *JSON object* | `-`          | The trusted certificate authority to use for validating certificates presented by devices of the tenant for authentication purposes. See [Trusted Certificate Authority Format]({{< relref "#trusted-ca-format" >}}) for a definition of the content model of the object. |
 | *adapters*               | *no*      | *array*       | `-`          | A list of configuration options valid for certain adapters only. The format of a configuration option is described here [Adapter Configuration Format]({{< relref "#adapter-configuration-format" >}}). **NB** If the element is provided then the list MUST NOT be empty. **NB** Only a single entry per *type* is allowed. If multiple entries for the same *type* are present it is handled as an error. **NB** If the element is omitted then all adapters are *enabled* in their default configuration. |
+| *limits*                 | *no*      | *JSON object* | `-`          | The limits such as the maximum number of connections can be set. The format of a configuration option is described here [Limits Configuration Format]({{< relref "#limits-configuration-format" >}}).|
```

Comment:
IMHO just `limits` is a little too unspecific for users to understand what it is about. What about `resource-limits`? Also, the type should be just *array* (analogous to field *adapters*).

Reply:
@sophokles73 `resource-limits` sounds fine to me. I will also update the `TenantConstants `accordingly. I didn't get it why it should be an _array_ analogous to field _adapters_. Each tenant configuration has only one `resource-limits` config as in below _example_ and not an _array_:

```
 {
    "tenant-id": "TENANT_TEST",
    "enabled": true,
    "resource-limits": {
      "max-connections": 25,
      "max-bytes-per-month": 26214400
    },
    "adapters": [
      {
        "type": "hono-http",
        "enabled": true,
        "device-authentication-required": true
      },
      {
        "type": "hono-mqtt",
        "enabled": false,
        "device-authentication-required": true
      }
    ]
  }
```

Reply:
my bad, should be an *object*, of course.